linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/rule.yml

```diff
@@ -9,7 +9,7 @@ description: |-
     {{{ xccdf_value("var_time_service_set_maxpoll") }}} in <tt>/etc/ntp.conf</tt> or
     <tt>/etc/chrony.conf</tt> to continuously poll time servers. To configure
     <tt>maxpoll</tt> in <tt>/etc/ntp.conf</tt> or <tt>/etc/chrony.conf</tt>
-    add the following:
+    add the following after each `server` entry:
```

Comment:
Since we are actually correctly accepting `pool` and `peer` properties as well in the OVAL, looks reasonable make this information explicit in the description.